### ISSUE TITLE
hax.cuts.apply_lichen reports lichen version as nan for some cuts

### ISSUE THREAD

**Issue by miwig** [XENON1T/hax#179]
apply_lichen [tries to get the lichen version from the uninstantiated class](https://github.com/XENON1T/hax/blob/39c82e1eb3bc64a88180a834268be8d95dcaa30f/hax/cuts.py#L227), but some lichens, like `S2AreaFractionTop` [only set their version when instantiated](https://github.com/XENON1T/lax/blob/c7d781de5e3f3c7df4b1595c31ac0e8a5301e98d/lax/lichens/sciencerun0.py#L540).

To reproduce:  
```
data = cuts.apply_lichen(data,'S2AreaFractionTop')
cuts.history(data)
```

**Comment by pdeperio** [comment]
@adambrown1 do we still need the 2 versions right now? or how about splitting it off into another cut like ```S2SingleScatterSimple``` does?

**Comment by adambrown1** [comment]
We probably no longer need two versions. But I think there is also a
trivial fix by moving the first pair of parentheses () from line 223 in
cuts.py to the end of line 220. I can make a pull request tomorrow or you
can change it now.


2017-12-13 17:58 GMT+01:00 Patrick de Perio <notifications@github.com>:

**Comment by JelleAalbers** [comment]
This should be fixed in #224
